cli/daemon/run/run.go

```diff
@@ -457,10 +467,18 @@ func (r *Run) generateConfig(p *Proc, params *startProcParams) *config.Runtime {
 		}
 	}
 	return &config.Runtime{
-		AppID:         r.ID,
-		AppSlug:       r.AppSlug,
+		AppID:   r.ID,
+		AppSlug: r.AppSlug,
+		APIBaseURL: url.URL{
+			Scheme: "http",
+			Host:   fmt.Sprintf("localhost:%d", params.RuntimePort),
+		},
+		DeployID:      fmt.Sprintf("clidpl_%s", xid.New()),
```

Comment:
We use `dpl` for deployments from our platform, so I've used `clidpl` to indicate this is a CLI deployment.

The reason this is set, rather than blank is so we more closely mimic a cloud-deployed application

Reply:
We didn't get a chance to discuss this one on Friday, but how about `run_%s` instead for local dev? `clidpl` is a bit terse (and a real consonant soup!)